### PR TITLE
Fix #2326: reduce amount of keyspaces created by ITs, add clean up

### LINKED ISSUE #2326: REST tests should not create a keyspace for each test method
* Create a keyspace at least for each test class instead
* Delete the created keypsace after test is done
* Tests could extend the `CqlEnabledIntegrationTestBase` so we have a single class that manages session and keyspace creation

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

1. Reduces number of keyspaces REST API Integration tests create, and
2. Cleans up keyspaces that area created

**Which issue(s) this PR fixes**:
Fixes #2326

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by tatu-at-datastax** [comment]
@ivansenic I think this should work to remove keyspace clutter; creates fewer keyspaces and cleans up ones that are created. Not super elegant but should be easy to further change if we want to as logic is mostly in base class.

File: apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaKeyspacesIT.java [hunk 1]
```diff
@@ -86,7 +85,8 @@ public void keyspacesGetAllMissingToken() {
   }
 
   // 09-Aug-2022, tatu: Alas, Auth token seems not to be checked
-  @Ignore("Auth token handling hard-coded, won't fail as expected")
+  // @Ignore("Auth token handling hard-coded, won't fail as expected")
```
Comment (ivansenic) [inline apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaKeyspacesIT.java:88]: remove commented out code?

**Review by ivansenic — APPROVED** [review]
Cool stuff..

File: apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaKeyspacesIT.java (hunk 1 above)
Comment (tatu-at-datastax) [reply apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaKeyspacesIT.java:88]: Ah yes. thanks!

**Pull request merged**